### PR TITLE
Feature/acs 1835 test dau for deleted renditions

### PR DESCRIPTION
**Pull request by mikolajbrzezinski**

### CODE REVIEW

File: remote-api/src/test/java/org/alfresco/rest/DeletedNodesTest.java [hunk 1]
```diff
@@ -47,6 +47,7 @@
 import org.alfresco.repo.content.MimetypeMap;
 import org.alfresco.rest.api.Nodes;
 import org.alfresco.rest.api.model.NodeTargetAssoc;
+import org.alfresco.rest.api.model.Site;
```
Comment (SaraAspery) [inline remote-api/src/test/java/org/alfresco/rest/DeletedNodesTest.java:50]: unused import

File: remote-api/src/test/java/org/alfresco/rest/DeletedNodesTest.java [hunk 2]
```diff
@@ -732,6 +736,45 @@ public void testRequestArchivedContentDirectUrl() throws Exception
...
+        File file = getResourceFile(fileName);
+        MultiPartBuilder multiPartBuilder = MultiPartBuilder.create().setFileData(new MultiPartBuilder.FileData(fileName, file));
+        MultiPartBuilder.MultiPartRequest reqBody = multiPartBuilder.build();
+
+        // Upload quick.pdf file into 'folder'
+        HttpResponse response = post(getNodeChildrenUrl(f1Id), reqBody.getBody(), null, reqBody.getContentType(), 201);
+        Document document = RestApiUtil.parseRestApiEntry(response.getJsonResponse(), Document.class);
+        String contentNodeId = document.getId();
+
+        Rendition rendition = createAndGetRendition(contentNodeId, "doclib");
```
Comment (SaraAspery) [inline remote-api/src/test/java/org/alfresco/rest/DeletedNodesTest.java:762]: can you just confirm that this method will wait for the rendition to be created?

File: remote-api/src/test/java/org/alfresco/rest/api/tests/AbstractBaseApiTest.java [hunk 3]
```diff
@@ -224,6 +224,11 @@ protected String getRequestArchivedContentDirectUrl(String nodeId)
         return URL_DELETED_NODES + "/" + nodeId + "/" + REQUEST_DIRECT_ACCESS_URL;
     }
 
+    protected String getRequestArchivedRenditonContentDirectUrl(String nodeId, String renditionID)
```
Comment (SaraAspery) [inline remote-api/src/test/java/org/alfresco/rest/api/tests/AbstractBaseApiTest.java:227]: This needs a rendition id in the endpoint request:
deleted-nodes/{{nodeId}}/renditions/{{renditionId}}/request-direct-access-url

File: remote-api/src/test/java/org/alfresco/rest/api/tests/AbstractBaseApiTest.java [hunk 4]
```diff
@@ -224,6 +224,11 @@ protected String getRequestArchivedContentDirectUrl(String nodeId)
...
+    {
+        return URL_DELETED_NODES + "/" + nodeId + "/" + URL_RENDITIONS + "/" + renditionID + "/" + REQUEST_DIRECT_ACCESS_URL;
```
Comment (SaraAspery) [inline remote-api/src/test/java/org/alfresco/rest/api/tests/AbstractBaseApiTest.java:229]: as above

**Review by SaraAspery — CHANGES_REQUESTED** [review]
Not an easy one to take on, looks good Mikolaj.
One small change needed :)

File: remote-api/src/test/java/org/alfresco/rest/api/tests/AbstractBaseApiTest.java (hunk 3 above)
Comment (SaraAspery) [reply remote-api/src/test/java/org/alfresco/rest/api/tests/AbstractBaseApiTest.java:227]: sorry, didn't spot that had already changed

File: remote-api/src/test/java/org/alfresco/rest/api/tests/AbstractBaseApiTest.java (hunk 4 above)
Comment (SaraAspery) [reply remote-api/src/test/java/org/alfresco/rest/api/tests/AbstractBaseApiTest.java:229]: as above

File: remote-api/src/test/java/org/alfresco/rest/DeletedNodesTest.java (hunk 2 above)
Comment (mikolajbrzezinski) [reply remote-api/src/test/java/org/alfresco/rest/DeletedNodesTest.java:762]: Yes it has, I hope its sufficient enough.

File: remote-api/src/test/java/org/alfresco/rest/DeletedNodesTest.java (hunk 1 above)
Comment (mikolajbrzezinski) [reply remote-api/src/test/java/org/alfresco/rest/DeletedNodesTest.java]: Done

**Pull request merged**